src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -867,19 +867,21 @@ Remove the '{0}' reference from your project and add the '{1}' NuGet package ins
 {0} - The missing tool name</comment>
   </data>
   <data name="XA5207" xml:space="preserve">
-    <value>Could not find android.jar for API level {0}. This means the Android SDK platform for API level {0} is not installed. Either install it in the Android SDK Manager ({2}), or change the Xamarin.Android project to target an API version that is installed. ({1} missing.)</value>
+    <value>Could not find android.jar for API level {0}. This means the Android SDK platform for API level {0} is not installed. {2} ({1} missing.) See https://aka.ms/xa5207 for more details.</value>
```

Comment:
@dellis1972: should we make this a multi-line error message?  It looks like it's going to be quite long.  Perhaps we want:

```xml
<value>Could not find android.jar for API level {0}. This means the Android SDK platform for API level {0} is not installed; it was expected to be in `{1}`.
{2}
See https://aka.ms/xa5207 for more details.</value>
```